BlazarUI/app/scripts/components/repo-build/RepoBuildDetail.jsx

```diff
@@ -111,16 +113,8 @@ class RepoBuildBuildDetail extends Component {
             build={build}
           />
         </div>
-        <div className='build-detail-body'>
-          <pre className='build-detail-body__commit-desc' title={currentCommit.message}>{truncate(currentCommit.message, this.state.windowWidth * .08, true)}</pre>
-        </div>
-        <div className='build-detail-footer'>
-          Triggered by <strong>{currentCommit.author.name}</strong> on { ' ' }
-          {timestampFormatted(currentCommit.timestamp, 'llll')} 
-          <span className='build-detail__sha'> 
-            commit <Sha gitInfo={gitInfo} build={build} />
-          </span>
-        </div>
+        <CommitsContainer
```

Comment:
I would suggest avoiding the use of `container` in the naming of this component for consistency in how `container` is used across the app. The app's current use of `container` is only used for a top level component that handle data fetching and state for its children (eg https://medium.com/@learnreact/container-components-c0e67432e005#.ifyto7j1n). That's not the case here, so I would suggest naming it something like `commits` and then renaming the current `commits` component to `commit-detail` or `commit-list` or something like that.